src/import-export.lisp

```diff
@@ -35,17 +35,11 @@
 (defun get-arg-list (fun-name)
   (let* ((signature-dict
           (pyeval "dict(inspect.signature(" fun-name ").parameters)")))
```

Comment:
`inspect.signature` is only available in 3.3+ https://docs.python.org/3/library/inspect.html#introspecting-callables-with-the-signature-object so this should probably either handle the error if `signature` doesn't exist, or use `inspect.getfullargspec` https://docs.python.org/3/library/inspect.html#inspect.getfullargspec

Reply:
Hmm... Will consider this. Thanks for bringing it to notice.

Reply:
I had `ignore-errors` while working with numpy: several callable objects do not have a signature(!) So, `ignore-errors` is still going to be necessary: perhaps we can add (yet another) switch, which tells whether or not to use `ignore-errors` while extracting the signature. Would [that](https://github.com/digikar99/py4cl/commit/236ffa07b65707972afd3797fbd9e1cbb52a22bb#diff-255309f12b39a6a52de754217f683d28) (keeping the code as-is) do?

In the future, (perhaps, I won't find the time to work on it though), one direction of development is: user-developers are provided with the option of specifying function details (arg-list, call-method) from a "config" file corresponding to the python module. (These config files may be stored in some centralised manner, so that the efforts of one user-developer can benefit others.) At that time, this function, is anyways going to need some modification.